### ISSUE TITLE
Launching SKSE64 through MO2 brings up the game's launcher. 

### ISSUE THREAD

**Issue by PepsiYourTeeth** [ModOrganizer2/modorganizer#414]
This has been going on for days. I originally installed the game in my C:/Games folder, like I've always done in the past. Everything is fine, then MO starts launching the game's launcher when I try to launch SKSE64 through Mod Organizer. 

Then I installed it in Program Files. Worked for a day, then same problem. 

Then I installed it directly onto my C drive. Worked for a day, then same problem.

Then I sent a MO shortcut to my desktop and made it run as administrator. Worked for a day, then same problem. 

I'm out of ideas. Thinking maybe I should go back to an earlier version since I've never (ever, in the past four years) had this happen to me. 

And MO isn't detecting my mods, of course, given that it's not even launching the game properly. 

Edit: putting Steam in offline mode worked. But whether it will continue to work is anyone's guess.

**Comment by Project579** [comment]
Maybe you have a program that is deleting MO configuration files?

**Comment by Silarn** [comment]
As the ticket framework suggests, we would need at least basic logs to have any clue what's going on. This doesn't sound quite like any issue I've heard of before.

**Comment by PepsiYourTeeth** [comment]
Sorry I'm so late getting back to you. Hopefully, the files attached correctly. I'm actually able to use Mod Organizer just fine if I keep Steam offline. 
[mo_interface.log](https://github.com/Modorganizer2/modorganizer/files/2152287/mo_interface.log)
[usvfs-2018-06-26_20-54-37.log](https://github.com/Modorganizer2/modorganizer/files/2152288/usvfs-2018-06-26_20-54-37.log)

**Comment by PepsiYourTeeth** [comment]
No one has an answer for this? I can run it with a shortcut from my desktop and it's okay, but I'd like to know why I can't run it from the taskbar without it asking to log into Steam and then pulling up the launcher. I'm trying to launch the game with SKSE64, by the way.

**Comment by mrfearless** [comment]
Did u run the launcher beforehand and set your options and then ran skyrim once? if not maybe your ini files are not created/default ini files and launcher is called to allow you to set them. Running it through MO with SKSE64 might force it to load launcher if its not seeing any ini files/using default ini files.

Might have to check in the ini editor to see that they the ini settings are correct as well.

Have steam open and running before launching SKSE64 (from either MO or outside it)

Try launching SKSE64 loader on its own outside of MO (MO closed) to verify its loading correctly - does it bring up launcher? or load the game. Then we can narrow down the possible issue.

**Comment by PepsiYourTeeth** [comment]
I did run the launcher and set my options beforehand like you said, yes. And if I run SKSE64_loader.exe from the root folder it brings up the game's launcher...whether Steam is open or not, offline or online. The only way I've been able to make this work is if I open MO2 from a desktop shortcut that's set to run as administrator. That way it doesn't matter if I have Steam online or offline or open or not, it starts the game correctly. If I try to run SKSE64 in any other way the game's launcher pops up. I don't even know if I consider this problem a problem anymore since I found a workaround. Mostly, I just want to bring it to everyone's attention.
